lib/node/zone.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var core = require('../core');
+var nodePatch = require('../patch/node');
+var longStackTraceZone = require('../zones/long-stack-trace');
+
+global.zone = new core.Zone();
+core.Zone.longStackTraceZone = longStackTraceZone;
+
+module.exports = {
```

Comment:
It would be nice if the exports here were `rootZone`, `bindPromiseFn`, and `longStackTraceZone`. Then there'd be no need to put them on the Zone ctor on node. But I imagine having the public APIs roughly equal for node and browser is important?